### PR TITLE
Fix wording of one fingerprinting test

### PR DESCRIPTION
**Pull request by stephendonner**
Fixes https://github.com/brave/qa-resources/issues/338

### CODE REVIEW

File: wikitemplate.md [hunk 1]
```diff
@@ -75,7 +75,7 @@
 - [ ] Visit https://jsfiddle.net/bkf50r8v/13/, ensure 3 blocked items are listed in Shields. Result window should show `got canvas fingerprint 0` and `got webgl fingerprint 00`
 - [ ] Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ only when `Block all fingerprinting protection` is on
 - [ ] Test that Brave browser isn't detected via user-agent, on https://www.whatismybrowser.com.  The site **will** say "Looks like Brave on [Windows/Linux/macOS]", and yet, "But it's announcing that it's Chrome [version] on [platform]"
-- [ ] Test that https://diafygi.github.io/webrtc-ips/ doesn't leak IP address when `Block all fingerprinting protection` is on
+- [ ] Test that https://diafygi.github.io/webrtc-ips/ doesn't leak IP address when `Fingerprinting blocked (strict, may break sites)` is on
```
Comment (srirambv) [inline wikitemplate.md:78]: Actually this test is wrong/outdated. WebRTC is not handled via `WebRTC IP handling policy` setting under Privacy and Security. More discussion [here](https://bravesoftware.slack.com/archives/C0816QH3R/p1618496627224900?thread_ts=1618487465.215300&cid=C0816QH3R) and [here](https://bravesoftware.slack.com/archives/C0816QH3R/p1618491867218700?thread_ts=1618487465.215300&cid=C0816QH3R). Also check these issues https://github.com/brave/brave-browser/issues/11310

This test should be changed to 

- [ ] Test that https://diafygi.github.io/webrtc-ips/ doesn't leak IP address for each option under `Settings -> Privacy and Security -> WebRTC IP handling policy`

**Review by srirambv — CHANGES_REQUESTED** [review]
This also needs to be updated for the Android template file so please make the change there as well

File: wikitemplate.md (hunk 1 above)
Comment (stephendonner) [reply wikitemplate.md]: Fixed.

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**